### PR TITLE
Ranger, your friendly K9 moral support!

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Adding in Ranger, a K9, near prep

## Why It's Good For The Game

More content. People like dog, and having a friendly K9 around prep for moral support adds to that.

RANGER, LEAD THE WAY!

BECOME BETTER THAN CM! HAVE DOGS FOR TGMC MARINES!!

Reused sprites from Halo Station 13 (https://github.com/HaloSpaceStation/HaloSpaceStation13/pull/2080)! Thank you for the awesome sprites! License under this baby: https://creativecommons.org/licenses/by-sa/3.0/

## Changelog
add: Ranger, your friendly K9 at prep! RANGER, LEAD THE WAY!
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
https://www.youtube.com/watch?v=edSpPKzc0zw

**Review by Kuro020 — APPROVED** [review]
add a picture of ranger on the comments so people can see

**Comment by novaepee** [comment]
image

**Comment by novaepee** [comment]
image

File: _maps/map_files/Minerva/TGS_Minerva.dmm [hunk 1]
```diff
@@ -6174,6 +6170,7 @@
 /obj/structure/cable,
 /obj/machinery/atmospherics/pipe/simple/supply/hidden/layer1,
 /obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,
+/mob/living/simple_animal/corgi/Ranger,
```
Comment (Mechanelize) [inline _maps/map_files/Minerva/TGS_Minerva.dmm:6173]: that is not a corgi

Comment (novaepee) [reply _maps/map_files/Minerva/TGS_Minerva.dmm:6173]: Pug is also put as corgi, which is the reason why I put Ranger under corgi.

File: code/modules/mob/living/simple_animal/friendly/corgi.dm [hunk 2]
```diff
@@ -17,14 +17,24 @@
...
-	real_name = "pug"
-	desc = "It's a pug."
-	icon_state = "pug"
-	icon_living = "pug"
-	icon_dead = "pug_dead"
+/mob/living/simple_animal/corgi/Ranger
+	name = "Ranger"
+	real_name = "Ranger"
+	gender = MALE
+	desc = "That's Ranger, your friendly k9. He has seen the terror of xenomorphs, so it's best to be nice to him. RANGER LEAD THE WAY!"
```
Comment (XSlayer300) [inline code/modules/mob/living/simple_animal/friendly/corgi.dm:34]:
```suggestion
	desc = "That's Ranger, your friendly and fierce K9 dog. He has seen the terror of Xenomorphs, so it's best to be nice to him. <b>RANGER LEAD THE WAY!</b>"
```

**Comment by BraveMole** [comment]
Cannot even take C4 and charge the xenos.... 0/10

File: code/modules/mob/living/simple_animal/friendly/corgi.dm (hunk 2 above)
Comment (Mechanelize) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:34]: It's just k9 not k9 dog

**Comment by Kuro020** [comment]
> Cannot even take C4 and charge the xenos.... 0/10

you monster

**Comment by BonkaiB** [comment]
Doggo :)

**Comment by novaepee** [comment]
> Cannot even take C4 and charge the xenos.... 0/10

soon:tm:

**Pull request merged**